test/functional/feature_init.py

```diff
@@ -33,20 +36,17 @@ def init_stress_test(self):
         - test terminating initialization after seeing a certain log line.
         - test removing certain essential files to test startup error paths.
         """
-        # TODO: skip Windows for now since it isn't clear how to SIGTERM.
-        #
-        # Windows doesn't support `process.terminate()`.
-        # and other approaches (like below) don't work:
-        #
-        #   os.kill(node.process.pid, signal.CTRL_C_EVENT)
-        if platform.system() == 'Windows':
-            raise SkipTest("can't SIGTERM on Windows")
-
         self.stop_node(0)
         node = self.nodes[0]
 
         def sigterm_node():
-            node.process.terminate()
+            if platform.system() == 'Windows':
+                # Don't call Python's terminate() since it calls
+                # TerminateProcess(), which unlike SIGTERM doesn't allow
+                # bitcoind to perform any shutdown logic.
+                os.kill(node.process.pid, signal.CTRL_BREAK_EVENT)
```

Comment:
Can this approach be used for all platforms?

Reply:
No. `CTRL_BREAK_EVENT` only exists on Windows: https://docs.python.org/3/library/signal.html#signal.CTRL_BREAK_EVENT.